library/core/class.controller.php

```diff
@@ -545,7 +545,7 @@ public function data($Path, $Default = '') {
     public function definitionList($wrap = true) {
         $Session = Gdn::session();
         if (!array_key_exists('TransportError', $this->_Definitions)) {
-            $this->_Definitions['TransportError'] = T('Transport error: %s', 'A fatal error occurred while processing the request.<br />The server returned the following response: %s');
+            $this->_Definitions['TransportError'] = T('Transport error: %s', 'A fatal error occurred while processing the request. The server returned the following response: %s');
```

Comment:
The <br> was probably there for a reason. It should be replaced by `\n` not space.